lighthouse-core/gather/driver.js

```diff
@@ -743,6 +745,45 @@ class Driver {
     };
   }
 
+  /**
+   * Return a promise that resolves when an insecure security state is encountered
+   * and a method to cancel internal listeners.
+   * @return {{promise: Promise<LH.Crdp.Security.SecurityStateExplanation[]>, cancel: function(): void}}
+   * @private
+   */
+  _waitForSecurityCheck() {
```

Comment:
rename since it resolves only in the case of *in*security (and the check isn't singular, it just goes until it isn't needed)? `_waitForInsecureState`? `_monitorForInsecureState`? Something else?